zigpy/quirks/v2/__init__.py

```diff
@@ -243,15 +243,28 @@ class EntityMetadata:
     cluster_type: ClusterType = attrs.field(default=ClusterType.Server)
     initially_disabled: bool = attrs.field(default=False)
     attribute_initialized_from_cache: bool = attrs.field(default=True)
-    translation_key: str = attrs.field()
-    fallback_name: str = attrs.field()
+    translation_key: str | None = attrs.field(default=None)
+    fallback_name: str = attrs.field(validator=attrs.validators.instance_of(str))
```

Comment:
I've used the in-built validation of `attrs` here. This is the message printed when `fallback_name` isn't provided:
```python
TypeError: ("'fallback_name' must be <class 'str'> (got None that is a <class 'NoneType'>)."
```

If wanted, I can also add it to our own validation below and add a clearer error message. I think the in-built one should be fine though.